index.bs

```diff
@@ -314,7 +314,7 @@ The following is an informative summary of the format specifiers processed by th
   <tr>
     <td>`%s`</td>
     <td>Element which substitutes is converted to a string</td>
-    <td><a abstract-op>ToString</a>(|element|)</td>
+    <td><a constructor>%String%</a>(|element|)</td>
```

Comment:
It looks like we can get rid of the [ToString](https://github.com/whatwg/console/blob/master/index.bs#L22) URL prefix now that we have no more references to it, if you'd like to remove that as well. Also feel free to put your name in the acknowledgements.